docs/book/supported_features_matrix.md

```diff
@@ -3,19 +3,19 @@
 # vSphere CSI Driver - Supported Features Matrix
 
 | | **Native K8s on vSphere 6.7U3 (CSI `1.0.2` & `2.0`)** | **Native K8s on vSphere 7.0 (CSI `2.0`)** | **vSphere with Tanzu – Supervisor Cluster** | **vSphere with Tanzu – TKG Service ‘Guest’ Cluster** |
-|---------------------------------------------------------|----------------------|------------------|--------------------|------------------|
-| CNS UI Support                                          | Yes                  | Yes              | Yes                | Yes              |
-| Enhanced Object Health in UI                            | Yes (_vSAN only_)    | Yes (_vSAN only_)| Yes (_vSAN only_)  | Yes (_vSAN only_)|
-| Dynamic Block PV support (`Read-Write-Once` Access Mode)| Yes                  | Yes              | Yes                | Yes              |
-| Dynamic File PV support (`Read-Write-Many` Access Mode) | No                   | Yes (_vSAN only_)| No                 | No               |
-| Encryption support via VMcrypt                          | No                   | Yes (_Block Volume_) | No                 | No               |
-| Dynamic Virtual Volume PV support                       | No                   | Yes              | No                 | No               |
-| Offline Volume Grow support (beta)                      | No                   | Yes (_Block Volume_) | No                 | No               |
-| Topology/Availability Zone support                      | Yes (_Block Volume_)     | Yes (_Block Volume_) | No                 | No               |
-| Static PV Provisioning                                  | Yes                  | Yes              | No                 | Yes              |
-| K8s Multi-node Control Plane support                    | Yes                  | Yes              | Yes                | Yes              |
-| `WaitForFirstConsumer`                                  | Yes                  | Yes              | No                 | No               |
-| VolumeHealth                                  | No                  | No              | Yes (_Block Volume_)                 | Yes (_Block Volume_)               |
+|---------------------------------------------------------|----------------------|----------------------|----------------------|----------------------|
```

Comment:
This matrix is specific to `v1.02` and `v2.0.0`. I think we need to discuss how to make it more generic so we can just add  to it for every new release.

Reply:
I created a different issue to update this table - https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/305